ds3_java_cli/src/main/java/com/spectralogic/ds3cli/exceptions/Ds3ExceptionHandler.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * *****************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ***************************************************************************
+ */
+
+package com.spectralogic.ds3cli.exceptions;
+
+public interface Ds3ExceptionHandler<Throwable> {
+
+    public void handle(final Throwable e);
+
+    // implemeters are encouraged to use format() as the message
+    // to facilitate testing
+    public String format(final Throwable e);
```

Comment:
I don't think this is needed since it's an internal call and not used by an external interface.